corehq/form_processor/backends/sql/dbaccessors.py

```diff
@@ -921,7 +921,7 @@ def check_transaction_order_for_case(case_id):
         with connections[db].cursor() as cursor:
             cursor.execute(
                 'SELECT compare_server_client_case_transaction_order(%s, %s)',
-                [case_id, CaseTransaction.case_rebuild_types()])
+                [case_id, CaseTransaction.case_rebuild_types() | CaseTransaction.TYPE_CASE_CREATE])
```

Comment:
For cases that haven't ever been rebuilt, there wasn't a "first transaction" to work from, which made one of the query assumptions fail on those cases. I sort of liked this solution over making it just take all transactions in that case, since it _should be_ the same thing, but commcare seems to have a way of handling if the create one comes in later than others that I wasn't sure I wanted to mess with.

Reply:
As long as there is a 'create' _somewhere_ it will check starting with that one. Is that a bad assumption?